kafka-ui-api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
-RUN apk add openjdk13-jre
+RUN apk add --no-cache openjdk13-jre libc6-compat gcompat
```

Comment:
What about maven dependencies mentioned in the issue?

Reply:
@Haarolean nothing connected with them - all of them already included in kafka-client lib. We just need to add missing os libs. I checked topic all compressions with this change - all work fine

Reply:
Great, thanks!